lib/ruby_lsp/requests/code_actions.rb

```diff
@@ -38,14 +38,19 @@ def initialize(document, range, context)
       def run
         diagnostics = @context[:diagnostics]
 
-        code_actions = diagnostics.filter_map do |diagnostic|
-          code_action = diagnostic.dig(:data, :code_action)
-          next if code_action.nil?
+        code_actions = diagnostics.flat_map do |diagnostic|
+          diagnostic_code_actions = diagnostic.dig(:data, :code_actions) || []
+          diagnostic_code_actions.filter_map do |code_action|
+            # We want to return only code actions that are within range or that do not have any edits, such as refactor
+            # code actions
+            range = code_action.dig(:edit, :documentChanges, 0, :edits, 0, :range)
+            unless cover?(range)
+              # Seems to work?
+              code_action[:title] += " RANGE?"
+            end
 
-          # We want to return only code actions that are within range or that do not have any edits, such as refactor
-          # code actions
-          range = code_action.dig(:edit, :documentChanges, 0, :edits, 0, :range)
-          code_action if diagnostic.dig(:data, :correctable) && cover?(range)
```

Comment:
we filter out the autocorrect actions upstream of this, since the rubocop comment actions are applicable even if the autocorrect isn't.